tests/draft_test.go

```diff
@@ -53,6 +53,7 @@ func TestDraftSavedAgain(t *testing.T) {
 		c.C("A002 SELECT Drafts").OK("A002")
 
 		_ = s.messageCreated("user", mailboxID, []byte("To: 3@3.pm"), time.Now())
+		s.flush("user")
```

Comment:
Not relevant to this PR but i wonder why/how you realized you had to add this? Did your changes affect the runtime behaviour in such a way as to require a flush? Or was it already flaky and you just hit a failure here at some point?

Reply:
This was a test that would occasionally fail in the past.